src/components/atoms/Button.tsx

```diff
@@ -0,0 +1,17 @@
+type props = {
```

Comment:
props is generally not a good name for types. use proper naming conventions. 
Usually types can use Pascal Case method eg. ButtonProps

Reply:
Okay sure